collections/app/model/Node.scala

```diff
@@ -22,8 +22,7 @@ object Node {
           // use the T at this level or an empty node to hold children
           Node(currentSlug, loop(children, thisFullPath), thisFullPath, thisLevel.headOption)
         }
-        .sortBy(_.fullPath.head)
-        .sortBy(_.children.isEmpty)
+        .sortBy(node => (node.children.isEmpty, node.basename))
```

Comment:
Ah OK, wasn't sure why the No children thing. Is it sorting recursively down the tree too?